config.go

```diff
@@ -130,6 +132,11 @@ type config struct {
 	DcrdServ         string `long:"dcrdserv" description:"Hostname/IP and port of dcrd RPC server to connect to (default localhost:9109, testnet: localhost:19109, simnet: localhost:19556)" env:"DCRDATA_DCRD_URL"`
 	DcrdCert         string `long:"dcrdcert" description:"File containing the dcrd certificate file" env:"DCRDATA_DCRD_CERT"`
 	DisableDaemonTLS bool   `long:"nodaemontls" description:"Disable TLS for the daemon RPC client -- NOTE: This is only allowed if the RPC client is connecting to localhost" env:"DCRDATA_DCRD_DISABLE_TLS"`
+
+	// ExchangeBot settings
+	DisableExchangeBot bool   `long:"no-exchange-monitor" description:"Disable the exchange monitor." env:"DCRDATA_NO_EXCHANGES"`
+	DisabledExchanges  string `long:"disable-exchange" description:"Exchanges to disable. Use a comma to separate multiple exchanges." env:"DCRDATA_DISABLE_EXCHANGES"`
```

Comment:
I'm thinking we should have a default setting for `DisabledExchanges`, perhaps set to `"dragonex,huobi"`.